app.js

```diff
@@ -1,131 +1,157 @@
 
-        alertInfo.innerText = '';
-        inputSetTime.value = '';
+          console.log(timeFromInput)
+          
+          panelDiv.appendChild(UrlLink);
+          panelDiv.appendChild(ourSetTime);
+          panelDiv.appendChild(deleteBtn);
 
 
+          alertInfo.innerText = '';
+          inputSetTime.value = '';
 
+          return panelDiv
+      } else {
+          alertInfo.innerText = " Alert ! Wpisz jakiś url i wybierz czas"
+      }
+  }
 
-    } else {
-        alertInfo.innerText = " Alert ! Wpisz jakiś url i wybierz czas"
-    }
-}
 
 
 
 
 
-//Usunięcie elementu z naszej listy 
+  //Usunięcie elementu z naszej listy 
 
-function deleteElement(e) {
-    // nasze e to event który przechwytamy , tutaj e to nasz Click / mouse event 
-    // sprawdzamy czy najbliższy button elementu w który kliknęliśmy zawiera klase btnDelete
-    if (e.target.closest('button').classList.contains('btnDelete')) {
-        console.log(e)
-        console.log(e.target)
-        console.log(e.target.closest('div'))
-        const deleteElement = e.target.closest('div')
-        deleteElement.remove();
-    }
-}
+  function deleteElement(e) {
+      // nasze e to event który przechwytamy , tutaj e to nasz Click / mouse event 
+      // sprawdzamy czy najbliższy button elementu w który kliknęliśmy zawiera klase btnDelete
+      if (e.target.closest('button').classList.contains('btnDelete')) {
+          console.log(e)
+          console.log(e.target)
+          console.log(e.target.closest('div'))
+          const deleteElement = e.target.closest('div')
+          deleteElement.remove();
+      }
+  }
 
 
 
-btnAdd.addEventListener('click', creatListElement);
-ulList.addEventListener('click', deleteElement);
+  btnAdd.addEventListener('click', animateDisplayItems);
+  ulList.addEventListener('click', deleteElement);
 
-// Timer 
+  // Timer 
 
-const btnStartCounter = document.querySelector('.btnStart');
-const btnStopCounter = document.querySelector('.btnStop');
-const btnResetCounter = document.querySelector('.btnReset');
-const counterTime = document.querySelector('.counter__time')
-let number = 0;
-let idSetInterval = false;
+  const btnStartCounter = document.querySelector('.btnStart');
+  const btnStopCounter = document.querySelector('.btnStop');
+  const btnResetCounter = document.querySelector('.btnReset');
+  const counterTime = document.querySelector('.counter__time')
+  let number = 0;
+  let idSetInterval = false;
 
-function startTimer() {
-    idSetInterval = true; // to blokuje  ponownie kliknac bo handleClick, zamieni sie na false i nie wchodzi do srodka wiec nie odpala nam ponownie startTimer()
-    idSetInterval = setInterval(() => {
-        number++;
-        counterTime.textContent = number;
-        console.log(number)
+  function startTimer() {
+      idSetInterval = true; // to blokuje  ponownie kliknac bo handleClick, zamieni sie na false i nie wchodzi do srodka wiec nie odpala nam ponownie startTimer()
```

Comment:
to raczej niepotrzebne :)

Reply:
masz rację , sprawdziłam działa poprawnie bez tego , a działa jeśli się nie mylę to dzięki  if w funkcji  handleClick , no przecież tam zachodzi zamiana na z true false , i tak jak tłumaczyłeś na pierwszej lekcji zachodzi to raz , w if zamiana z false na true , i to true nie przechodzi dalej bo my tu nie nadpisujemy wartości w tym if .